src/CommunityVoices/App/Website/Presentation/Module/BatchUploadConfirm.xslt

```diff
@@ -259,14 +259,17 @@
                         </div>
                     </div>
                 </xsl:if>
-                <div id="entryIssues">
+                <div id="entryIssues" style="display: none;">
```

Comment:
@Sammidysam Why did you make this change? I believe that this is being toggled as appropriate in the jQuery

Reply:
It is indeed being toggled as appropriate, but it starts out as being displayed even if there are no errors.  I feel like it makes more sense to display nothing then only show it if we find errors?  We could also just show "Loading..." in the box or something until errors are populated or not.

Reply:
@Sammidysam Hmm ... interesting. When I tried to do this on a sheet with no errors the box didn't display for me. I think it will only display if loading? 

As long as the box ends up displaying once errors occur then this should be fine though. I can test this.

Reply:
Resolved over phone call - we deemed this fine.  My JavaScript is slower than yours, I guess.